modules/core/02-client/client/cli/tx.go

```diff
@@ -287,10 +288,6 @@ func NewCmdSubmitUpdateClientProposal() *cobra.Command {
 				return err
 			}
 
-			if err = msg.ValidateBasic(); err != nil {
```

Comment:
validatebasic still exists on some messages, its up to the chain to decide if they want to remove or not. for that reason we should keep it here

```go
if m, ok := msg.(sdk.HasValidateBasic); ok {
...
}
```

that is what would be needed to replace it

Reply:
**very helpful**

Reply:
The msg type in reference is not an interface (concrete type) and doesn't have validate basic so it can be safely removed